### DISCUSSION TITLE
`onCellEditComplete` warning in DataTable

### DISCUSSION THREAD

**Discussion by annette-graham** [primefaces/community#1025] [PrimeReact] [answered]
Hi, I am getting the following error in the DataTable component upon render:

`Warning: Unknown event handler property onCellEditComplete. It will be ignored.`

As per the example in the documentation, I have the `onCellEditComplete={onCellEditComplete}` prop on the `Column` component.
Is there any reason I am getting this? 

Thanks!

**Comment by melloware** [comment]
@annette-graham I can't reproduce: https://stackblitz.com/edit/6rur8p?file=src%2FApp.jsx  No issues I can see or warnings using PR 10.3.3

**Reply by annette-graham** [reply]
@melloware thanks for trying all the same!

**Reply by annette-graham** [reply, marked as answer]
It seems that deleting and reinstalling my `node_modules`  and `yarn.lock` resolved the issue. I might have had an older version lurking or something..
